### DISCUSSION TITLE
JBang with Spring Native

### DISCUSSION THREAD

**Discussion by linux-china** [jbangdev/jbang#1235] [Show and tell]
Prototype ready, and you can create GraalVM native-image app with Spring Native. 

![Snip20220209_86](https://user-images.githubusercontent.com/46711/153363352-bc6ed694-d30c-4f2e-bd3c-806d183715ef.png)

**Comment by quintesse** [comment]
Nice! 👍

**Comment by maxandersen** [comment]
cool -got a link to its source ?

**Reply by linux-china** [reply]
@maxandersen  how about create repo under jbangdev, such as jbangdev/spring-native？   I know some developers like Quarkus native, but still lots of developers use Spring and Spring native, and me too :) 

It's not hard, just call a Picocli command supplied by Spring AOT.

**Reply by maxandersen** [reply]
I don't think jbangdev should host/be directly connected to any specific external runtime integration if/when it can be avoided. 

I suggest you start it somewhere on your personal account and work it up and ask spring org will host it which would be a much more sustainable place to have it.
 
Happy to link and document to it in our docs as examples just like we do for Quarkus.
